src/providers/twitch/TwitchChannel.cpp

```diff
@@ -405,6 +440,36 @@ void TwitchChannel::sendMessage(const QString &message)
     }
 }
 
+void TwitchChannel::sendReply(const QString &message, const QString &replyId)
+{
+    auto app = getApp();
+    if (!app->accounts->twitch.isLoggedIn())
+    {
+        if (!message.isEmpty())
+        {
+            this->showLoginMessage();
+        }
+
+        return;
+    }
+
+    qCDebug(chatterinoTwitch) << "[TwitchChannel" << this->getName()
+                              << "] Send reply message:" << message;
+
+    // Do last message processing
+    QString parsedMessage = this->prepareMessage(app, message);
+
+    bool messageSent = false;
+    this->sendReplySignal.invoke(this->getName(), parsedMessage, replyId,
+                                 messageSent);
+
+    if (messageSent)
+    {
+        qCDebug(chatterinoTwitch) << "sent";
```

Comment:
Do you mean just the `qCDebug` call? I had kept it because we do the same logging in `TwitchChannel::sendMessage`, but we could remove both if we don't think we need it.

Reply:
Yes, I was referring to the debug output. Fair point regarding the use in `::sendMessage()`, I wasn't aware when making the comment. I think it could be removed in both places then but I would like to hear other opinions on that as well.